nixos/modules/programs/spacefm.nix

```diff
@@ -0,0 +1,41 @@
+# Global configuration for spacefm.
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.spacefm;
+
+in
+{
+  ###### interface
+
+  options = {
+
+    programs.spacefm = {
+
+      settings = mkOption {
+        type = types.attrs;
+        default = {};
+        example = literalExample ''{
```

Comment:
Ok looking at https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/display-managers/lightdm.nix I think it's possible if I'll use an enable bool. I'll look up how to setup a vm\container to test it since so far I was just abusing travis to do my module testing :D